src/Service.Host/client/src/components/PurchaseOrders/CreatePurchaseOrderUt.js

```diff
@@ -278,6 +270,23 @@ function CreatePurchaseOrderUt() {
         }
     }, [previousOrderResults]);
 
+    useEffect(() => {
+        if (partSuppliersSearchResults?.length) {
+            handleSupplierChange({
+                id: `${partSuppliersSearchResults.find(s => s.supplierRanking === 1).supplierId}`
+            });
+            dispatch({
+                payload: {
+                    lineNumber: 1,
+                    fieldName: 'ourUnitPriceCurrency',
+                    value: partSuppliersSearchResults.find(s => s.supplierRanking === 1)
+                        .currencyUnitPrice
+                },
+                type: 'detailFieldUpdate'
+            });
+        }
+    }, [detail.id, partSuppliersSearchResults]);
```

Comment:
can you remove detail.id from this array? It doesn't seem to be used in the effect

Reply:
sorted.